.vscode/tasks.json

```diff
@@ -31,4 +32,4 @@
 			"problemMatcher": []
 		}
 	],
-}
+}
```

Comment:
add back trailing line break.

Reply:
Will do.  Note that the VS Code formatter is removing this line break at the end of the file.